packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -110,6 +110,10 @@ RocketChat.statistics.get = function _getStatistics() {
 		platform: process.env.DEPLOY_PLATFORM || 'selfinstall',
 	};
 
+	statistics.uploadsTotal = RocketChat.models.Uploads.find().count();
+	const [result] = await RocketChat.models.Uploads.model.rawCollection().aggregate([{ $group: { _id: 'total', total: { $sum: '$size' } } }]).toArray();
```

Comment:
```suggestion
	const [result] = Promise.await(RocketChat.models.Uploads.model.rawCollection().aggregate([{ $group: { _id: 'total', total: { $sum: '$size' } } }]).toArray());
```

you can use `Promise.await` here so you don't need to change every place where this function is called.

`Promise.await` will encapsulate the promise in a Fiber and wait for it to resolve. pretty handy when you have code already relying in the Fiber.